sidebars.ts

```diff
@@ -13,7 +13,9 @@ const sidebars: SidebarsConfig = {
                 type: 'category', label: "Before You Begin", items: [
                     'intro/before-you-begin/setting-up-flutterflow', 'intro/before-you-begin/app-architecture', 'intro/quickstart'
                 ]
-            }],
+            },{type: 'category', label: "FlutterFlow UI", items: [
+                'intro/ff-ui/builder', 'intro/ff-ui/navigation-menu','intro/ff-ui/widget-palette','intro/ff-ui/storyboard','intro/ff-ui/toolbar','intro/ff-ui/canvas','intro/ff-ui/resource-hierarchy'
+            ]}],
```

Comment:
I think the hierarchy is wrong here. Its 

- Getting Started 
- Quickstart 
- Before Your Begin
- Understanding the FF UI 

But in your case, quickstart, before you begin, ff-ui all are nested within Getting Started. 
Please correct this and also you don't have to list pages manually for the non-intro directories. See how the other directories are set up here.